sbatches/farmshare/singularity-jupyter.sbatch

```diff
@@ -15,19 +15,21 @@
 # $ bash start.sh singularity /path/to/container <args>
 
 PORT=$1
-NOTEBOOK_DIR=${2:-${SCRATCH}}
-CONTAINER=${3:-docker://vanessa/repo2docker}
+NOTEBOOK_DIR=/farmshare/scratch/users/$USER
+CONTAINER=/farmshare/home/classes/bioe/301p/ce/ce
 
-export SINGULARITY_CACHEDIR=/farmshare/user_data/${USERNAME}/.singularity
+export SINGULARITY_CACHEDIR=/farmshare/user_data/${USERNAME}
 echo "Container is ${CONTAINER}"
 echo "Notebook directory is ${NOTEBOOK_DIR}"
 cd ${NOTEBOOK_DIR}
 
-# Create .local folder for default modules, if doesn't exist
+#Create .local folder for default modules, if doesn't exist
 if [ ! -d "${HOME}/.local" ];
   then
     echo "Creating local python modules folder to map at ${HOME}/.local";
     mkdir -p "${HOME}/.local";
 fi
 
-singularity exec --home ${HOME} --bind ${HOME}/.local:/home/username/.local ${CONTAINER} jupyter notebook --no-browser --port=$PORT --ip 0.0.0.0
+. /etc/profile
+module load singularity/3.4.0
+singularity exec /farmshare/home/classes/bioe/301p/ce/ces jupyter notebook --no-browser --port=$PORT --ip 0.0.0.0
```

Comment:
This should be a reference to `${CONTAINER}`

Reply:
Also I think you need to add the bind to $HOME/.local back, especially if you want installs to work. Just verify the container uses /home/username

Reply:
Fixed on 173023f